invenio_communities/assets/semantic-ui/js/invenio_communities/settings/privileges.js

```diff
@@ -36,18 +37,23 @@ class CommunityPrivilegesForm extends Component {
     this.setState({ error: error.response.data.message });
   };
 
+  setIsSavedState = (newValue) => {
+    this.setState({isSaved: newValue})
+  }
+
   render() {
+    const { isSaved } = this.state
     return (
       <Formik
         initialValues={this.getInitialValues(this.props.community)}
         onSubmit={async (
           values,
-          { setSubmitting, setErrors, setFieldError }
+          { setSubmitting, setErrors, setFieldError, setIsActive }
         ) => {
           setSubmitting(true);
           const payload = values;
           try {
-            await axios.put(
+            axios.put(
```

Comment:
Use `await` here :) You don't need the `.then()` as that syntax is used for Promises...